### PR TITLE
refactor and add github actions

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

File: config/config-infra-dev.yml [hunk 1]
```diff
@@ -1,36 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-variables:
```
Comment (nellyk) [inline config/config-infra-dev.yml:4]: replaced by the file in .workflows/variables.yml

File: infrastructure/pipelines/bicep-ado-deploy-infra.yml [hunk 2]
```diff
@@ -1,76 +0,0 @@
...
-          deploy:
-            steps:
-            - checkout: self
-            - task: AzureCLI@2
-              displayName: Running ${{ variables.environment }} Deployment
-              inputs:
-                azureSubscription: $(ado_service_connection_rg)
-                scriptType: bash
-                scriptLocation: inlineScript
-                inlineScript: |
```
Comment (nellyk) [inline infrastructure/pipelines/bicep-ado-deploy-infra.yml:69]: This is all Azure Devops

File: mlops/devops-pipelines/deploy-model-training-pipeline.yml [hunk 3]
```diff
@@ -1,60 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-variables:
-# - ${{ if eq(variables['Build.SourceBranchName'], 'main') }}:
```
Comment (nellyk) [inline mlops/devops-pipelines/deploy-model-training-pipeline.yml:5]: All Azure Devops files

File: templates/aml-cli-v2/connect-to-workspace.yml [hunk 4]
```diff
@@ -1,12 +0,0 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-steps:
-  - task: AzureCLI@2
-    displayName: Connect to AML Workspace using CLI v2
```
Comment (nellyk) [inline templates/aml-cli-v2/connect-to-workspace.yml:6]: all azure devops files

File: templates/infra/create-storage-container.yml [hunk 5]
```diff
@@ -1,11 +0,0 @@
...
-# Licensed under the MIT License.
-
-steps:
-  - task: AzureCLI@2
-    displayName: 'Create Container'
-    inputs:
-      azureSubscription: '$(ado_service_connection_rg)'
-      scriptType: 'bash'
-      scriptLocation: 'inlineScript'
-      inlineScript: 'az storage container create --account-name $(terraform_st_storage_account) --name $(terraform_st_container_name)'
```
Comment (nellyk) [inline templates/infra/create-storage-container.yml:11]: all azure devops files

**Review by yodobrin — APPROVED** [review]

**Pull request merged**